### PR TITLE
MTA-3367 RHSSO admin info

### PR DESCRIPTION
**Pull request by vsodhi**
JIRA: https://issues.redhat.com/browse/MTA-3367

### CODE REVIEW

File: docs/topics/mta-7-installing-web-console-on-openshift.adoc [hunk 1]
```diff
@@ -233,17 +233,24 @@ The amount of memory available for running pods on this node is 28.9 GiB. This a
 [id="mta-7-red-hat-single-sign-on_{context}"]
 == Red Hat Single Sign-On
 
-{ProductShortName} delegates authentication and authorization to a
-https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.6[Red
-Hat Single Sign-On] (RHSSO) instance managed by the {ProductShortName} operator. Aside from controlling the full lifecycle of the managed RHSSO instance, the {ProductShortName} operator also manages the configuration of a dedicated
-https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.6/html/server_administration_guide/configuring_realms[realm] that contains all the roles and permissions that {ProductShortName} requires.
+The {ProductShortName} uses link:https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.6[Red
+Hat Single Sign-On] (RHSSO) instance for user authentication and authorization.
```
Comment (anarnold97) [inline docs/topics/mta-7-installing-web-console-on-openshift.adoc:237]:
```suggestion
The {ProductShortName} uses link:https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.6[Red Hat Single Sign-On (RHSSO)] instance for user authentication and authorization.
```

File: docs/topics/mta-7-installing-web-console-on-openshift.adoc [hunk 2]
```diff
@@ -233,17 +233,24 @@ The amount of memory available for running pods on this node is 28.9 GiB. This a
...
-For more information, see
-https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.6/html/server_administration_guide/red_hat_single_sign_on_features_and_concepts[Red Hat Single Sign-On features and concepts].
+The admin credentials for RHSSO are stored in a secret file named `credential-mta-rhsso` in the namespace where {ProductShortName} is installed.
+
+To retrieve your admin credentials, run the following command:
+----
+oc get secret credential-mta-rhsso -o yaml
+----
+
+To create a dedicated route for the RHSSO instance, set the `rhsso_external_access` parameter to `True` in the Tackle CR for {ProductShortName}.
```
Comment (anarnold97) [inline docs/topics/mta-7-installing-web-console-on-openshift.adoc:250]:
```suggestion
To create a dedicated route for the RHSSO instance, set the `rhsso_external_access` parameter to `true` in the Tackle custom resource (CR) for {ProductShortName}.
```

Comment (vsodhi) [reply docs/topics/mta-7-installing-web-console-on-openshift.adoc]: fixed.

File: docs/topics/mta-7-installing-web-console-on-openshift.adoc (hunk 1 above)
Comment (vsodhi) [reply docs/topics/mta-7-installing-web-console-on-openshift.adoc]: fixed.

**Review by rromannissen — CHANGES_REQUESTED** [review]
Rewrite doesn't cover the requested changes.

**Comment by vsodhi** [comment]
@rromannissen Based on our discussion, I've added the missing link under the "additional resources" subsection; please ack. Thank you!

**Pull request merged**